src/vse_sync_pp/parsers/parser.py

```diff
@@ -3,7 +3,46 @@
 """Common parser functionality"""
 
 import json
-from decimal import Decimal
+import re
+from datetime import (datetime, timezone)
+from decimal import (Decimal, InvalidOperation)
+
+# sufficient regex to extract the whole decimal fraction part
+RE_ISO8601_DECFRAC = re.compile(
+    r'^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.(\d+)'
+)
+
+def parse_timestamp_abs(val):
+    """Return a :class:`Decimal` from `val`, an absolute timestamp string.
+
+    Accepted absolute timestamp strings are ISO 8601 format with restrictions.
+    The string must: explicitly specify UTC timezone, supply time in seconds,
+    specify a decimal fractional part with a decimal mark of '.'.
+
+    Return None if `val` is not a string or is not an ISO 8601 format string.
+    Raise :class:`ValueError` otherwise.
+    """
+    try:
+        dtv = datetime.fromisoformat(val)
```

Comment:
That one is not working for python 3.10. Waiting for update or add as requirement 3.11 (wfm).